lib/beam/src/Obelisk/Database/Beam/Entity.hs

```diff
@@ -59,6 +65,32 @@ deriving instance Ord (Columnar f k) => Ord (Id k f)
 deriving instance Show (Columnar f k) => Show (Id k f)
 deriving instance Read (Columnar f k) => Read (Id k f)
 
+fromEK
+  :: KeyT value ~ Id k
+  => EntityKeyT f value -> Columnar f k
+fromEK (EntityKey (Id x)) = x
```

Comment:
I sort of honestly prefer to just define the `Iso` as `entityIdKey` or something like that and only use the lens combinators.

Reply:
Makes sense. Though, `entityIdKey` or `entityKeyId` ?